ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -953,18 +964,71 @@ impl fmt::Display for DisplayableOperand<'_> {
 }
 
 /// A constant.
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+///
+/// Note that this struct deliberately does not implement `PartialEq` (or `Eq`): two instances of
+/// `Const` may represent the same underlying constant, but (because of floats), you as the user
+/// need to determine what notion of equality you wish to use on a given const.
+#[derive(Clone, Debug)]
 pub(crate) enum Const {
     /// A constant integer at most 64 bits wide. This can be treated a signed or unsigned integer
     /// depending on the operations that use this constant (the [Ty::Integer] type itself has no
     /// concept of signedness).
+    #[allow(dead_code)]
```

Comment:
I suppose this `#[dead_code]` annotation can go now.

Reply:
Fixed in https://github.com/ykjit/yk/pull/1267/commits/c96852d70914bf6e35bd7c9ddabf2be19b4a9f30